viewing_party/party.py

```diff
@@ -1,23 +1,112 @@
 # ------------- WAVE 1 --------------------
 
+
 def create_movie(title, genre, rating):
-    pass
+    return (
+        {"title": title, "genre": genre, "rating": rating}
+        if title and genre and rating
+        else None
+    )
+
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+
+def get_movie_from_watchlist(user_data, movie_title):
+    return next((m for m in user_data["watchlist"] if m["title"] == movie_title), None)

```

Comment:
This is pretty succinct and made me look up the `next` function! I think it might be a bit too succinct for my style tastes, but that's fine, it fits in with what generally seems to be your style.

Reply:
Oh, I didn't even notice that this is a helper function you added until I looked at the next one! Good work on doing that to make your other code cleaner.